### ISSUE TITLE
[BUG] Search Pop-up (CTRL-F) doesn't remember size.

### ISSUE THREAD

**Issue by wangsuki** [notepad-plus-plus/notepad-plus-plus#15442]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description of the Issue

The Search Pop-up (CTRL-F) doesn't remember the horizontal size. Every time I open Notepad++ and open the Search Pop-up it reverts to default size.

### Steps To Reproduce

1. Open Notepad++.
2. Open Search Pop-up (CTRL-F or Search -> Find...).
3. Drag the Search Pop-up longer horizontally.
4. Close Notepad++ (Exit, not minimize).
5. Open Notepad++.
6. Open Search Pop-up (CTRL-F or Search -> Find...).

### Current Behavior

The Search Pop-up (CTRL-F) opens as default size whenever a new instance of Notepad++ opens.

### Expected Behavior

The Search Pop-up should remember the last horizontal size every time a new instance of Notepad++ opens.

### Debug Information

```shell
Notepad++ v8.6.8   (64-bit)
Build time : Jun  4 2024 - 00:30:00
Path : C:\Program Files\Notepad++\notepad++.exe
Command Line : 
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
Periodic Backup : OFF
OS Name : Windows 11 Home (64-bit)
OS Version : 23H2
OS Build : 22631.3880
Current ANSI codepage : 1252
Plugins : 
    ComparePlus (1.2)
    Explorer (1.9.9)
    mimeTools (3.1)
    NppConverter (4.6)
    NppExport (0.4)
```

### Anything else?

![Search Pop-up - default size](https://github.com/user-attachments/assets/de09bbee-db61-4fb0-aa88-a89f90e2f51a)
![Search Pop-up - long size](https://github.com/user-attachments/assets/98dc3f50-11a9-4331-b0dd-f0bd3c99b254)

**Comment by vlakoff** [comment]
It should be fixed in Notepad++ 8.6.9, see #15294.

**Comment by ArkadiuszMichalski** [comment]
Duplicate of https://github.com/notepad-plus-plus/notepad-plus-plus/issues/15294.
